gateway-manager/src/manage_keycloak.py

```diff
@@ -154,6 +213,9 @@ def create_user(
         if _status_pwd:
             LOGGER.warning(f'- {str(_status_pwd)}')
     LOGGER.success(f'Added user "{user}" to realm "{realm}"')
+    if admin:
+        LOGGER.info(f'Granting user "{user}" admin rights on realm "{realm}"...')
+        assign_all_client_roles(realm, user, 'realm-management')
```

Comment:
Is `realm-management` a hardcoded value? Could it be different? Could it be a setting?

Reply:
It's hard-coded in keycloak and as far as I know will always be the client that controls who has access to the realm level controls.